SU2_CFD/src/solver_direct_mean.cpp

```diff
@@ -1038,7 +1052,25 @@ CEulerSolver::~CEulerSolver(void) {
     }
     delete [] SlidingStateNodes;
   }
-  
+
```

Comment:
You can move this to the dtor of CSolver too

Reply:
I just did, although that meant I needed to move the nMarker and nVertex declarations also there (no big deal). That's sorted as well then.